config/features.yml

```
@@ -695,6 +695,10 @@ features:
     actor_type: user
     description: enables confirmation modal when removing toxic exposure data from Form 526
     enable_in_development: true
+  disability_compensation_toxic_exposure_opt_out_data_purge:
+    actor_type: user
+    description: enables function that removes toxic exposure data if user has opted out form condition on Form 526 submission
```

Not required, I don't think, but many flippers are sorted alphabetically in each group. There's also a small typo in the description.
```suggestion
    description: enables function that removes toxic exposure data if user has opted out from condition on Form 526 submission
```